edn_tags.go

```diff
@@ -58,6 +58,14 @@ func (tm *TagMap) AddTagFn(tagname string, fn interface{}) error {
 	return tm.addVal(tagname, rfn)
 }
 
+// MustAddTagFn adds fn as a converter function for tagname tags to this TagMap
+// like AddTagFn, except this function panics if the tag could not be added.
+func (tm *TagMap) MustAddTagFn(tagname string, fn interface{}) {
+	if err := tm.AddTagFn(tagname, fn); err != nil {
+		panic(err.Error())
```

Comment:
I'd prefer the panic to return the error itself, so that people can inspect it if they (for some reason..) want to recover. It should be printed similarly I think. But if it looks bad without `.Error()` then I'm fine with leaving it in.

Reply:
Didn't know `panic` supported arbitrary arguments. I've made the change now :+1:.